knowledge-base/sigstore/cosign-installer/action-security.yml

```diff
@@ -0,0 +1,8 @@
+name: 'cosign-installer'
+outbound-endpoints:
+  - fqdn: 'storage.googleapis.com'
+    port: 443
+    reason: to fetch data 
```

Comment:
This is to download cosign, not fetch data
https://github.com/sigstore/cosign-installer/blob/47b61722b2c3f34f9f15f98f28a779f13cfb59cf/action.yml#L197